.github/workflows/e2e.workflow.yml

```diff
@@ -2,6 +2,9 @@
 name: e2e
 
 on:
+  # Allow invokable workflow run
+  workflow_dispatch:
```

Comment:
Just curious what `workflow_dispatch` does? Is this for running on-demand invocation?

Reply:
Yeah exactly. So we can verify it works without waiting for it being scheduled.